staticDocs/Permissions.md

```diff
@@ -0,0 +1,151 @@
+
+## Endpoints and their Required Roles
+
+### Approvals
+
+| Endpoint                                      | Permission | Permission Level |
+| --------------------------------------------- | ---------- | ---------------- |
+| getFormApprovalFlowInstance()                 | FaaS       | Developer        |
+| getFormSubmissionApproval()                   | FaaS       | Developer        |
+| searchFormSubmissionAdministrationApprovals() | FaaS       | Developer        |
+
+### Data Manager
+
+| Endpoint            | Permission       | Permission Level   |
+| ------------------- | ---------------- | ------------------ |
+| getFormDefinition() | Form Submissions | Manager, Read Only |
+| searchRecords()     | Form Submissions | Manager, Read Only |
+
+### Email Templates
+
+| Endpoint              | Permission      | Permission Level   |
+| --------------------- | --------------- | ------------------ |
+| createEmailTemplate() | Email Templates | Manager            |
+| deleteEmailTemplate() | Email Templates | Manager            |
+| getEmailTemplate()    | Email Templates | Manager, Read Only |
+| searchEmailTemplate() | Email Templates | Manager, Read Only |
+| updateEmailTemplate() | Email Templates | Manager            |
+
+### Form Element Lists
+
+| Endpoint                 | Permission         | Permission Level |
+| ------------------------ | ------------------ | ---------------- |
+| createFormElementList()  | Form Element Lists | Manager          |
+| deleteFormElementList()  | Form Element Lists | Manager          |
+| searchFormElementLists() | ---                | ---              |
+| updateFormElementList()  | Form Element Lists | Manager          |
+
+### Form Element Lookups
+
+| Endpoint                   | Permission           | Permission Level   |
+| -------------------------- | -------------------- | ------------------ |
+| createFormElementLookup()  | Form Element Lookups | Manager            |
+| deleteFormElementLookup()  | Form Element Lookups | Manager            |
+| getFormElementLookup()     | Form Element Lookups | Manager, Read Only |
+| searchFormElementLookups() | ---                  | ---                |
+| updateFormElementLookup()  | Form Element Lookups | Manager            |
+
+### Forms
+
+| Endpoint                          | Permission       | Permission Level   |
+| --------------------------------- | ---------------- | ------------------ |
+| createForm()                      | Forms            | Manager            |
+| createSubmissionAttachment()      | FaaS             | Developer          |
+| deleteForm()                      | Forms            | Manager            |
+| executeWorkflowEvent()            | Form Submissions | Manager            |
+| generateFormUrl()                 | Forms\*          | Manager, Read Only |
+|                                   | Apps             | Manager, Read Only |
+|                                   | FaaS\*\*         | Developer          |
```

Comment:
This is not only required for prefill data upload, it will also be required to actually submit the form.